app/models/models.go

```diff
@@ -455,3 +455,21 @@ type PurgeQueueRequest struct {
 }
 
 func (r *PurgeQueueRequest) SetAttributesFromForm(values url.Values) {}
+
+/*** SNS */
+
+func NewCreateTopicRequest() *CreateTopicRequest {
+	return &CreateTopicRequest{}
+}
+
+// Ref: https://docs.aws.amazon.com/sns/latest/api/API_CreateTopic.html
+type CreateTopicRequest struct {
+	Name string `json:"Name" schema:"Name"`
+
+	// Goaws unsupports below properties currently.
+	DataProtectionPolicy string            `json:"DataProtectionPolicy" schema:"DataProtectionPolicy"`
+	Attributes           map[string]string `json:"Attributes" schema:"Attributes"`
```

Comment:
I understand the current code isn't doing anything with them, but let's grab these by name at least.  Checkout the `CreateQueueV1` work for an example of how to handle those attributes

Reply:
Updated!